resources/monitoring/templates/kyma-additions/alertmanager.config.yaml

```diff
@@ -30,6 +34,9 @@ route:
     continue: true # If set to `false`, it stops after the first matching.
     match_re:
       {{ tpl ( toYaml .Values.global.alertTools.credentials.slack.matchExpression | indent 6) . }}
+    {{- with .Values.global.alertTools.credentials.slack.routes }}
```

Comment:
Why you are not using the same approach as in the match_re by not using the "with" function but only "tpl" without "with"? Any specific reason?

Reply:
Hello @a-thaler @hisarbalik , finally i try to reuse the {{ with by taking an example from  the kyma/resources/dex/templates/dex-config-map.yaml， and later connector : | could support multiple lines....
```
{{- with .Values.connectors }}
    connectors:
    {{- tpl . $ | nindent 4 }}
  {{- end }}
```
could u help to take a review? thanks.